datashape/discovery.py

```diff
@@ -15,7 +15,7 @@
                         Record, string, Null, DataShape, real, date_, time_,
                         Unit, timedelta_, TimeDelta, object_, String)
 from .predicates import isdimension, isrecord
-from .py2help import _strtypes, _inttypes
+from .py2help import _strtypes, _inttypes, MappingProxyType
```

Comment:
I actually like having the custom 2/3 lib. This code is pretty simple and we have already added a bunch of our own stuff like OrderedDict and now mapping proxy so idk what we would be saving.